source/dub/commandline.d

```diff
@@ -599,6 +599,7 @@ abstract class PackageBuildCommand : Command {
 		bool m_forceRemove = false;
 		bool m_single;
 		bool m_filterVersions = false;
+		bool m_noSaveSelections;
```

Comment:
nit, but imo, using positives is easier to parse (gets rid of having to deal with double negatives)

Reply:
(i.e. making the var `saveSelections` as opposed to `noSaveSelections`)

Reply:
done:)